src/main/resources/assets/btweagles/loot_tables/fish_heads.json

```diff
@@ -0,0 +1,74 @@
+{
+    "pools": [
+        {
+            "name": "btweagles:fish_heads",
+            "rolls": 1,
+            "entries": [
+                {
+                    "type": "item",
+                    "name": "minecraft:skull",
+                    "entryName": "btweagels:fish_head",
```

Comment:
The mod ID "btweagles" is misspelled in this file. Other than that I think the PR is fine.